shared/components/Animation/__tests__/Reveal.spec.jsx

```diff
@@ -1,26 +1,37 @@
 import React from 'react'
-import { render } from 'enzyme'
+import { mount } from 'enzyme'
 
 import Reveal from '../Reveal'
 
 describe('Reveal', () => {
-  it('renders when the transition is triggered', () => {
-    const reveal = render(
-      <Reveal timeout={500} in={true} height={300}>
+  const doMount = (overrides = {}) =>
+    mount(
+      <Reveal {...overrides} timeout={500} height={300}>
```

Comment:
Small thing here. The overrides should be applied AFTER the defaults. Otherwise they are not really overrides. :)